### ISSUE TITLE
Develop New Icons

### ISSUE THREAD

**Issue by Kcaden20** [Kcaden20/critical-code-cookbook#26]
Develop icons for new categories 
![Screen Shot 2022-06-24 at 1 22 52 PM](https://user-images.githubusercontent.com/13590895/175611348-3c00f15d-eb75-488a-ae86-43c60fe435e6.png)

**Comment by Kcaden20** [comment]
@xinemata new icons have been made and even better, the custom SVG input worked like a charm! Let me know if we're good with the icons.
![Screen Shot 2022-06-27 at 2 17 32 PM](https://user-images.githubusercontent.com/13590895/176009122-e05ba983-b47d-43ce-a0a0-453c5a66a99c.png)

**Comment by xinemata** [comment]
Very fun! Would it be possible to make Python's icon a little more different from p5? I think now that everything else looks so customized the similarity between those two stands out to me.

**Comment by Kcaden20** [comment]
@xinemata Yeah we can. I think the idea originally was when they were our only language options that it'd be fun for them to be mirrors of one another (a 5 and a snake) but can I'll customize given the changes since.

**Comment by Kcaden20** [comment]
@xinemata new python snake
![Screen Shot 2022-06-27 at 2 35 50 PM](https://user-images.githubusercontent.com/13590895/176011858-6b762bbc-58c7-46d5-a7dd-ae2cdd992d8d.png)

**Comment by xinemata** [comment]
Fun! Looks good to me.

**Comment by xinemata** [comment]
Working on both desktop & mobile on my end. Are we describing these shapes with aria-labels?

**Comment by Kcaden20** [comment]
@xinemata will work on adding in `aria-labels`

**Comment by xinemata** [comment]
Breakfast icon: “abstract line drawing of a bowl”
Lunch icon: “abstract line drawing of three intersecting plates”
Dinner icon: “abstract line drawing of a serving bowl, intersected by four plates”
p5 icon: “abstract line drawing of the number 5”
Code agnostic icon: “abstract line drawing of the earth”
JavaScript icon: “abstract line drawing of the letter J”
Processing icon: “abstract line drawing of the letter P”
Python icon: “abstract line drawing of the python snake”
SQL icon: “abstract line drawing of a barrel”

**Comment by Kcaden20** [comment]
Added these in using `titles` as described in this article. Also added the `role="img"` so different browsers don't read them differently. Missing from this list was also the icons for the chapter sections.

https://css-tricks.com/accessible-svgs/

I described those in similar terms to the language for the other icons.  
![Screen Shot 2022-06-30 at 8 04 54 PM](https://user-images.githubusercontent.com/13590895/176797779-5b0666cf-02eb-4cb5-b491-d794b472e9bb.png)

**Comment by xinemata** [comment]
This looks great. Closing issue.
